person.rb

```diff
@@ -0,0 +1,24 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  # @param [String] name, [Number] age
```

Comment:
Kindly consider removing this line since leaving old pieces of code as inline comments will maky your project look messy and and bulky as the project size increases. If you need to review a previous version of your code, you can always use git history. :pray: